auto_submit/test/service/validation_service_test.dart

```diff
@@ -533,19 +545,11 @@ void main() {
         retryOptions: const RetryOptions(delayFactor: Duration.zero, maxDelay: Duration.zero, maxAttempts: 3),
       );
 
-      // Create a result that will trigger a retry.
-      QueryResult queryResultException = createFakeQueryResult(
-        exception: OperationException(
-          graphqlErrors: [
-            const GraphQLError(message: 'Base branch was modified. Review and try the merge again'),
-          ],
-        ),
-      );
-
-      QueryResult queryResultSuccess = createFakeQueryResult();
-      githubGraphQLClient.useMutationMapOnMutate = true;
-      githubGraphQLClient.mutationMap.add(queryResultException);
-      githubGraphQLClient.mutationMap.add(queryResultSuccess);
+      githubService.useMergeRequestMockList = true;
+      githubService.pullRequestMergeMockList
+          .add(PullRequestMerge(merged: false, message: 'Unable to merge pull request.'));
```

Comment:
nit: trailing commas would make this easier to read

Reply:
Oh man I was not aware that this also helps the formatter format the code! CooL!